apps/laboratory/tests/shared/utils/project.ts

```diff
@@ -2,7 +2,7 @@ import { devices } from '@playwright/test'
 
 import { DESKTOP_DEVICES, MOBILE_DEVICES } from '../constants/devices'
 
-const LIBRARIES = ['ethers', 'ethers5', 'wagmi', 'solana'] as const
+const LIBRARIES = ['ethers', 'ethers5', 'wagmi', 'solana', 'bitcoin'] as const
```

Comment:
don't we have another directive that only runs `wallet.spec` for btc tests?

Reply:
Line 72

Reply:
We have but we don't enable the bitcoin at all, so those configs in L72 will be used if `bitcoin` library tests run